packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -45,10 +50,10 @@ describe('Bootstrapping the application', () => {
       class FooComponent {
         providers = {foo: FooProvider};
       }
-
-      const app = new Application({components: [FooComponent]});
-
-      const value = app.getSync('foo');
+      const app = new Application();
+      const server = new RestServer(app, {components: [FooComponent]});
```

Comment:
Ditto, the component should be registered on `app` and this test should stay in `core`. There are few more tests down below that are subject to this comment too, please review them yourself.